packages/components/src/Combobox/components/ComboboxContent/ComboboxContent.tsx

```diff
@@ -19,17 +20,33 @@ interface ComboboxContentProps {
    * actions
    */
   readonly children: ReactElement | ReactElement[];
+
+  /**
+   * pre selected option
+   * @default ""
+   * @optional
+   * @type string
+   * */
+  readonly preSelectedOption?: string;
 }
 
 export function ComboboxContent(props: ComboboxContentProps): JSX.Element {
+  const { preSelectedOption } = props;
   const [searchValue, setSearchValue] = useState<string>("");
-  const [selectedOption, setSelectedOption] = useState<string>("");
+  const [selectedOption, setSelectedOption] = useState<string>(
+    preSelectedOption || "",
+  );
   const { open } = React.useContext(ComboboxContext);
 
   const filteredOptions = props.options.filter(option =>
     option.toLowerCase().includes(searchValue.toLowerCase()),
   );
+  let invalidPreSelectedOption = false;
+  if (preSelectedOption && !props.options.includes(preSelectedOption)) {
```

Comment:
I think this could be reduced down to a one liner

```
 const shouldGetUpset: boolean =
    !!props.preSelectedOption &&
    !props.options.includes(props.preSelectedOption);
```

Reply:
yup! great suggestion!